### ISSUE TITLE
Can't set up: > Configuration with name 'snobotSimLibs' not found.

### ISSUE THREAD

**Issue by calebeby** [ArcticWarriors/snobot-2018#46]
Hi,

I am trying to set up the sim for java on Linux, and followed the gradle directions in the wiki.

When I open eclipse to run the sim for the first time, `com.snobot.simulator` does not appear

When I run `./gradlew simulate`, it says:

```
Eclipse not configured, will not update classpath
:simulate FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':simulate'.
> Configuration with name 'snobotSimLibs' not found.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 0s
1 actionable task: 1 executed
```

Any ideas?

Thanks

**Comment by pjreiniger** [comment]
Are you trying to run our robot, our run the simulator with your robot?

**Comment by calebeby** [comment]
My robot

**Comment by pjreiniger** [comment]
Ok, wrong project but no big deal.  

So it sounds like you are using gradleRio, as we were on this project.  I didn't figure out how to get a command line option for running the simulator to work during the season, so you just need to run the simulator from Eclipse (or whatever IDE, the process should be similar).

The missing part is likely around step five [here](https://github.com/pjreiniger/SnobotSim/wiki/Setting-Up-The-Simulator#option-1---gradlerio).  If it doesn't see the simulators Main class (or any main classes for that matter) check and make sure that eclipse has synced up with the gradle scripts, and you see the simulator dependencies in the project.

![image](https://user-images.githubusercontent.com/10381694/39193314-41fc1176-47a9-11e8-8b80-638e23f84f5b.png)

**Comment by calebeby** [comment]
Yeah, sorry I posted in the wrong repo :joy:

I got the default simulator to open, I didn't realize I was supposed to right click on the project _in_ referenced libraries, I just thought I was supposed to click on the root of the overall project.

So now it works fine, except that I still can't launch the sim from gradle. Is there any way this is possible? I really don't like having to launch eclipse to run the sim.

Thanks for making this really cool simulator!
